vta/python/vta/build_module.py

```diff
@@ -115,3 +118,60 @@ def build(*args, **kwargs):
         with build_config():
             return tvm.build(*args, **kwargs)
     return tvm.build(*args, **kwargs)
+
+
+def vta_autotvm_build_func(measure_input, tmp_dir, **kwargs):
```

Comment:
Can we put a todo to unify all the build modules? it doesn't make sense to have one per platform